CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+
+## How to contribute
+
+🎈 You made it here! This is a great step in helping to contribute to ARK Navigator. 
+
+To get started, you can start off here [issues](https://github.com/ARK-Builders/ARK-Navigator/issues) with those tagged "good first issue" 
+
+The projects embeds ktlint. You can check it out in GitHub Actions workflow.
+
+## Creating a Pull Request / Pull Request Rules
+
+Before a PR can be merged, you would have to fix all code style errors.
```

Comment:
"We also care a lot about our software quality, that's why we are conducting code review before merging. Right now, the team isn't that big, so please be patient."

Reply:
Thanks for this, it has been updated